string configFile
---
aerialcore_msgs/FlightPlan[] plans